src/main/java/org/example/objectrepo/OrangeHRM_ObjRepo.java

```diff
@@ -596,6 +596,55 @@
     @FindBy(id = "assignBtn")
     public WebElement assignLeaveButton;
 
+    //Time PAge
```

Comment:
I saw many xpath you used in this class, they all look like default xpath that you just simply copied and pasted. Default xpath and CSS are not recommended. If you want to use them, you gotta make your own xpath and css. Please update them all.

Reply:
got it. I will update them. 
Thanks for the comment.